dace/config_schema.yml

```diff
@@ -132,7 +112,7 @@ required:
 
             allow_shadowing:
                 type: str
-                default: false 
+                default: true 
```

Comment:
Is this related, or did it just sneak in here? :-)

Reply:
Snuck in, can revert but I think it should be in master until the new codegen appears